sds_data_manager/lambda_code/SDSCode/api_lambdas/upload_api.py

```diff
@@ -152,5 +161,7 @@ def lambda_handler(event, context):
     s3_key_path_str = str(
         s3_key_path.relative_to(imap_data_access.config["DATA_DIR"]).as_posix()
     )
-
-    return _generate_signed_upload_response(s3_key_path_str)
+    s3tags = {"manually_reprocessed": "true"} if manually_reprocessed else None
+    if s3tags:
+        logger.info(f"Uploading file with S3 tags: {s3tags}.")
```

Comment:
Do we need this log on every upload? I assume this was for debugging. Or maybe we should log the tags/path inside the generate_signed_upload_response call.